docs/user-guide/introspection-and-debugging.md

```diff
@@ -35,53 +35,67 @@ spec:
         - containerPort: 80
 ```
 
+Copy this to a file *./my-nginx-dep.yaml*
```

Comment:
Alternatively we can use `cat <<EOF > /path/to/file` like http://kubernetes.io/docs/admin/resource-quota/#viewing-and-setting-quotas did. Which one do you think is better?